.github/workflows/create-release-without-prow.yaml

```diff
@@ -133,10 +141,10 @@ jobs:
       build-args: BIN=subaccount-sync
       tags: ${{ inputs.name }}
 
-  run-keb-chart-tests:
+  run-keb-chart-install-tests:
     name: Validate KEB chart 
     needs: [build-keb-image, build-archiver-image, build-environments-cleanup-image, build-deprovision-retrigger-image, build-expirator-image, build-runtime-reconciler-image, build-subaccount-cleanup-image, build-subaccount-sync-image]
-    uses: "./.github/workflows/run-keb-chart-tests-reusable.yaml"
+    uses: "./.github/workflows/run-keb-chart-install-tests-reusable.yaml"
```

Comment:
Is `run-keb-chart-tests-reusable.yaml` still used somewhere?

Reply:
In the current release workflow where we are using prow.

Reply:
And in tests on PR's.

Reply:
This is just a temporary for created for test convenience.